### DISCUSSION TITLE
attached files

### DISCUSSION THREAD

**Discussion by Larbo53** [jsvine/pdfplumber#986] [Ask for help with specific PDFs]
Hello,
is it possible to extract pdf attachments with pdfplumber? for example an xml file.
Thank you.
Sincerely

Facture_CARAT_FA00019_20221118-1.pdf

**Comment by jsvine** [comment]
Hi @Larbo53, can you attach an example PDF to your comment? I see the text `Facture_CARAT_FA00019_20221118-1.pdf`, but it is not linked to any file.

**Comment by Larbo53** [comment]
Hi,

Herewith.


Yves Larbodiere
EVARD
3, rue des Courtes Terres
95220 HERBLAY
portable : 07 81 08 41 00
mail : ***@***.***

**Reply by jsvine** [reply]
Hi @Larbo53, it looks like you may be trying to comment/upload via email. It appears this is not working. Could you try via GitHub's website directly?

**Comment by Larbo53** [comment]
[2FV 00001178 CAFE VAREN Sep23.pdf](https://github.com/jsvine/pdfplumber/files/12576447/2FV.00001178.CAFE.VAREN.Sep23.pdf)

**Reply by jsvine** [reply]
Thank you, @Larbo53. This link worked for me. I don't see an XML attachment within the PDF, however. Where do you see it appearing? (Or perhaps I misunderstand the original question?)

**Comment by Larbo53** [comment]
with printscreen



[printscreen.pdf](https://github.com/jsvine/pdfplumber/files/12576601/printscreen.pdf)

**Reply by jsvine** [reply]
This appears to be a PDF of a screenshot of an email. It seems that the *email* has an XML attachment, but the PDF itself (regardless of the tool used) will contain the XML information.

**Comment by Larbo53** [comment]
it's true. But with another pdf file it's different. These files are used for eletronic invoice in EU.

[EXTENDED_Rechnungskorrektur.pdf](https://github.com/jsvine/pdfplumber/files/12578619/EXTENDED_Rechnungskorrektur.pdf)
